FruitFruit/FruitFruit/Views/InitStoryboard/FruitTextField.swift

```diff
@@ -8,14 +8,15 @@
 import Foundation
 import UIKit
 
-class FruitTextField: UITextField {
-    var bottomBorder = UIView()    
+class FruitTextField: UITextField, UITextFieldDelegate {
+    var bottomBorder = UIView()
```

Comment:
```suggestion
    var bottomBorderView = UIView()
```

UI Component인지 알기쉽도록 이름이 명확해지면 좋겠네요 ~